src/CMakeLists.txt

```diff
@@ -51,23 +51,32 @@ if(ENABLE_CUDA OR ENABLE_OPENACC)
   set(GPU_FLAGS -gpu=${GPU_ARCH},lineinfo -Minfo=accel)
   message(STATUS "GPU Flags: ${OpenACC_CXX_FLAGS} ${GPU_FLAGS}")
 
-  # target_compile_options(micm_gpu PRIVATE ${OpenACC_CXX_FLAGS} ${GPU_FLAGS})
+  target_compile_options(micm_openacc PRIVATE ${OpenACC_CXX_FLAGS} ${GPU_FLAGS})
 
-  target_link_libraries(micm_gpu 
+  target_link_libraries(micm_openacc 
     PRIVATE micm
   )
 
-  if(ENABLE_CUDA)
-    target_link_libraries(micm_gpu 
-      PUBLIC cudart
-    )
-  endif()
-
   if(ENABLE_OPENACC)
```

Comment:
Is this `if(ENABLE_OPENACC)` redundant since it is already in a `if(ENABLE_OPENACC)` statemenet?

Reply:
Yes, fixed.